### PR TITLE
Update release testing instructions for windows

### PR DESCRIPTION
**Pull request by adriaanm-da**
CHANGELOG_BEGIN
CHANGELOG_END

### Pull Request Checklist

- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/main/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description
- [ ] If you mean to change the status of a component, please make sure you keep [the Component Status page](https://github.com/digital-asset/daml/blob/main/docs/source/support/component-statuses.rst) up to date.

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

**Review by stefanobaghino-da — APPROVED** [review]

**Review by SamirTalwar-DA — APPROVED** [review]

File: release/RELEASE.md [hunk 1]
```diff
@@ -76,9 +76,11 @@ patches we backport to the 1.0 release branch).
    >
    > [ad-hoc]: https://github.com/DACH-NY/daml-language-ad-hoc
    >
-   > `ad-hoc.sh` prints IP address, username and password for the created Windows VM.
+   > `ad-hoc.sh create windows` prints IP address, username and password for the created Windows VM.
    > Save this output. You will need this information later when you create an RDP connection.
-   >
+   > 
+   > ‼️ After connecting, it's going to take some time for the machine to be configured (see notes below).
```
Comment (garyverhaegen-da) [inline release/RELEASE.md:82]: The configuration of the machine starts immediately on startup, it doesn't wait for you to connect.

So if you create the machine, wait for fifteen minutes, and then connect, it should be ready to go. (Most of the time. Maybe plan for 20 minutes if you want to be sure.)

File: release/RELEASE.md [hunk 2]
```diff
@@ -90,15 +92,13 @@ patches we backport to the 1.0 release branch).
...
+   > NOTE 1: **Use Firefox for testing.** Windows machines come with both Internet Explorer and Firefox installed. Do not make the mistake of trying to use Internet Explorer.
    >
    > Ad-hoc machines also come with Node, VSCode and OpenJDK preinstalled, so
    > you don't need to worry about those.
    >
-   > The script that installs Firefox, Node, VSCode and OpenJDK runs once the
-   > machine is available for login. If you can't find the software you need
-   > immediately, just wait for a couple of minutes.
+   > NOTE 2: After logging in, **it takes some time for the machine to be configured.** The script that installs Firefox, Node, VSCode and OpenJDK runs once the
+   > machine is available for login. The software you need should appear within about 10 minutes.
```
Comment (garyverhaegen-da) [inline release/RELEASE.md:101]: A good way to check if the setup is over is to look for the `D:\` drive, as it is created after all the software is installed.

File: release/RELEASE.md (hunk 1 above)
Comment (stefanobaghino-da) [reply release/RELEASE.md:82]: Valid point, thanks for the good catch!

Comment (adriaanm-da) [inline release/RELEASE.md:82]:
```suggestion
   > ‼️ After starting, it's going to take some time for the machine to be configured (see notes below).
```

File: release/RELEASE.md (hunk 2 above)
Comment (adriaanm-da) [inline release/RELEASE.md:101]:
```suggestion
   > machine is available for login. The software you need should appear within about 10 minutes. (An easy way to check is to try to open `D:\` , as it is created after all the software is installed.)
```

**Review by garyverhaegen-da — APPROVED** [review]

**Pull request merged**